[2024-09-29T14:18:29.046+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-29T14:18:29.101+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.create_dataset scheduled__2024-07-09T00:00:00+00:00 [queued]>
[2024-09-29T14:18:29.130+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.create_dataset scheduled__2024-07-09T00:00:00+00:00 [queued]>
[2024-09-29T14:18:29.131+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-09-29T14:18:29.531+0000] {taskinstance.py:2327} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_dataset> on 2024-07-09 00:00:00+00:00
[2024-09-29T14:18:29.589+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl', 'create_dataset', 'scheduled__2024-07-09T00:00:00+00:00', '--job-id', '1060', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmplvpt0g74']
[2024-09-29T14:18:29.593+0000] {standard_task_runner.py:91} INFO - Job 1060: Subtask create_dataset
[2024-09-29T14:18:29.603+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=14016) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-29T14:18:29.606+0000] {standard_task_runner.py:63} INFO - Started process 14136 to run task
[2024-09-29T14:18:29.861+0000] {task_command.py:426} INFO - Running <TaskInstance: etl.create_dataset scheduled__2024-07-09T00:00:00+00:00 [running]> on host 71321b93f95e
[2024-09-29T14:18:30.173+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='create_dataset' AIRFLOW_CTX_EXECUTION_DATE='2024-07-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-09T00:00:00+00:00'
[2024-09-29T14:18:30.174+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-29T14:18:30.205+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-09-29T14:18:30.225+0000] {base.py:84} INFO - Using connection ID 'my-gcp-conn' for task execution.
[2024-09-29T14:18:30.451+0000] {bigquery.py:470} INFO - datasetId was not specified in `dataset_reference`. Will use default value finalproject_data.
[2024-09-29T14:18:30.462+0000] {bigquery.py:470} INFO - projectId was not specified in `dataset_reference`. Will use default value madt-finalproject.
[2024-09-29T14:18:30.463+0000] {bigquery.py:481} INFO - Creating dataset: finalproject_data in project: madt-finalproject 
[2024-09-29T14:18:31.225+0000] {bigquery.py:2045} INFO - Dataset finalproject_data already exists.
[2024-09-29T14:18:31.340+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-29T14:18:31.385+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=etl, task_id=create_dataset, execution_date=20240709T000000, start_date=20240929T141829, end_date=20240929T141831
[2024-09-29T14:18:32.412+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-09-29T14:18:32.478+0000] {taskinstance.py:3482} INFO - 9 downstream tasks scheduled from follow-on schedule check
[2024-09-29T14:18:32.488+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
